src/app/shared/mocks/notifications.mock.ts

```diff
@@ -0,0 +1,1872 @@
+import { of as observableOf } from 'rxjs';
+import { ResourceType } from '../../core/shared/resource-type';
+import {
+  QualityAssuranceTopicObject
+} from '../../core/suggestion-notifications/qa/models/quality-assurance-topic.model';
+import {
+  QualityAssuranceEventObject
+} from '../../core/suggestion-notifications/qa/models/quality-assurance-event.model';
+import {
+  QualityAssuranceTopicDataService
+} from '../../core/suggestion-notifications/qa/topics/quality-assurance-topic-data.service';
+import {
+  QualityAssuranceEventDataService
+} from '../../core/suggestion-notifications/qa/events/quality-assurance-event-data.service';
+import { DSpaceObject } from '../../core/shared/dspace-object.model';
+import { Item } from '../../core/shared/item.model';
+import {
+  createNoContentRemoteDataObject$,
+  createSuccessfulRemoteDataObject,
+  createSuccessfulRemoteDataObject$
+} from '../remote-data.utils';
+import { SearchResult } from '../search/models/search-result.model';
+import {
+  QualityAssuranceSourceObject
+} from '../../core/suggestion-notifications/qa/models/quality-assurance-source.model';
+
+// REST Mock ---------------------------------------------------------------------
+// -------------------------------------------------------------------------------
+
+// Items
+// -------------------------------------------------------------------------------
+
+const ItemMockPid1: Item = Object.assign(
+  new Item(),
+  {
+    handle: '10077/21486',
+    lastModified: '2017-04-24T19:44:08.178+0000',
+    isArchived: true,
+    isDiscoverable: true,
+    isWithdrawn: false,
+    _links:{
+      self: {
+        href: 'https://rest.api/rest/api/core/items/0ec7ff22-f211-40ab-a69e-c819b0b1f357'
+      }
+    },
+    id: 'ITEM4567-e89b-12d3-a456-426614174001',
+    uuid: 'ITEM4567-e89b-12d3-a456-426614174001',
+    type: 'item',
+    metadata: {
+      'dc.creator': [
+        {
+          language: 'en_US',
+          value: 'Doe, Jane'
+        }
+      ],
+      'dc.date.accessioned': [
+        {
+          language: null,
+          value: '1650-06-26T19:58:25Z'
+        }
+      ],
+      'dc.date.available': [
+        {
+          language: null,
+          value: '1650-06-26T19:58:25Z'
+        }
+      ],
+      'dc.date.issued': [
+        {
+          language: null,
+          value: '1650-06-26'
+        }
+      ],
+      'dc.identifier.issn': [
+        {
+          language: 'en_US',
+          value: '123456789'
+        }
+      ],
+      'dc.identifier.uri': [
+        {
+          language: null,
+          value: 'http://dspace7.4science.it/xmlui/handle/10673/6'
```

Comment:
Silly thing, but I'd really like to make all URLs generic and avoid referencing old XMLUI.  There are several in this file which start with "http://dspace7.4science.it/xmlui/".  Could we change these all to start instead with either `https://demo.dspace.org` or a fake URL like `https://example.org`